package.json

```
@@ -15,7 +15,9 @@
     "next": "13.2.3",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "styled-components": "^5.3.8"
+    "react-hook-form": "^7.43.5",
+    "styled-components": "^5.3.8",
+    "use-local-storage-state": "^18.2.0"
```

```suggestion
    "styled-components": "^5.3.8",
```
If you don't use certain packages at the moment, they shouldn't be part of a feature/PR